community.json

```diff
@@ -1395,9 +1395,9 @@
     },
     "shsd": {
         "branch": "master",
-        "level": 0,
+        "level": 3,
```

Comment:
Thank you for your contribution, but please do not change level manually ... this is meant to be set automatically from CI results. Otherwise this number can't be trusted :/.

Reply:
Thanks !

![jgpa3qa](https://user-images.githubusercontent.com/4533074/49892399-2dd80680-fe49-11e8-9973-97b84faac4d9.gif)